mjidris/irondb
# Us 294

## Objective
Fix a variety of bugs found as console warnings. 

## How to Test (if applicable)
There should be no change to functionality. The only tests required are to make sure that I didn't break anything. 

## Requirements
https://tree.taiga.io/project/cbrobsto-iron-db/us/294?milestone=258286[Taiga Task]
